README.md

```diff
@@ -4,45 +4,98 @@
 
 # Open3D: A Modern Library for 3D Data Processing
 
-<h4>
-    <a href="http://www.open3d.org">open3d.org</a> |
-    <a href="http://www.open3d.org/docs">Documentation</a> |
-    <a href="http://www.open3d.org/docs/release/getting_started.html">Quick Start</a> |
-    <a href="http://www.open3d.org/docs/release/compilation.html">Build from Source</a> |
-    <a href="http://www.open3d.org/docs/release/index.html#python-api-index">Python API</a> |
-    <a href="http://www.open3d.org/docs/release/cpp_api/index.html">C++ API</a> |
-    <a href="http://www.open3d.org/docs/release/contribute.html">Contribute</a> |
-    <a href="https://www.youtube.com/watch?v=I3UjXlA4IsU">Demo</a> |
-    <a href="https://forum.open3d.org">Forum</a>
-</h4>
+**[Homepage](http://www.open3d.org") |**
+**[Docs](http://www.open3d.org/docs") |**
+**[Viewer App](https://github.com/intel-isl/Open3D/releases") |**
+**[Quick Start](http://www.open3d.org/docs/release/getting_started.html") |**
+**[Build from Source](http://www.open3d.org/docs/release/compilation.html") |**
+**[Python API](http://www.open3d.org/docs/release/index.html#python-api-index") |**
+**[C++ API](http://www.open3d.org/docs/release/cpp_api/index.html") |**
+**[Contribute](http://www.open3d.org/docs/release/contribute.html") |**
+**[Demo](https://www.youtube.com/watch?v=I3UjXlA4IsU") |**
+**[Forum](https://forum.open3d.org")**
```

Comment:
all the URLs end with " but do not have " at the start

Reply:
fixed